dismob/log.py

```
@@ -143,32 +148,25 @@ async def safe_send_message(channel: discord.TextChannel, content: str = None, e
         error(f"Error when sending message: {e}")
         return None
 
-async def safe_respond(interaction: discord.Interaction, content: str = None, embed: discord.Embed = None, ephemeral: bool = False):
+async def safe_respond(interaction: discord.Interaction, content: str = None, embed: discord.Embed = None, view: discord.ui.View = None, file: discord.File = None, ephemeral: bool = False):
```

Ruff (0.14.1)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)